app.rb

```diff
@@ -0,0 +1,44 @@
+require 'sinatra/base'
```

Comment:
I would suggest creating a test database when you have the time in order to test functionality